skimage/restoration/_denoise.py

```diff
@@ -994,10 +993,10 @@ def estimate_sigma(image, average_sigmas=False, multichannel=False, *,
             sigmas = np.mean(sigmas)
         return sigmas
     elif image.shape[-1] <= 4:
-        msg = ("image is size {0} on the last axis, but channel_axis is "
-               "None.  If this is a color image, please set channel_axis=-1 "
-               "for proper noise estimation.")
-        warn(msg.format(image.shape[-1]))
+        msg = f'image is size {image.shape[-1]} on the last axis, '\
+              f'but channel_axis is None. If this is a color image, '\
+              f'please set channel_axis=-1 for proper noise estimation.'
```

Comment:
```suggestion
              'please set channel_axis=-1 for proper noise estimation.'
```

I think i prefer parenthesis to blackslashes. but that is rather minor.